core/src/main/scala/io/funktional/rumpel/Dictionary.scala

```diff
@@ -15,12 +15,16 @@ trait Dictionary:
      * @return a tuple containing the updated Random instance and the picked word
      */
     def pickOne(random: Random): (Random, String)
+
+    def separator(separator: String): Dictionary =
+        Dictionary.Filled(separator, this)
```

Comment:
Where is this method used? I don't see why we're having it. Is it to handle words with spaces?